DiscordChatExporter.Domain/Exporting/Partitioners/IPartitioner.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DiscordChatExporter.Domain.Exporting
+{
+    interface IPartitioner
+    {
+        bool IsLimitReached(long messageCount, long sizeInBytes);
```

Comment:
Can we make it `bool IsLimitReached(ExportPartitioningContext context)` and move the properties to a class?